src/features/profile/Profile.tsx

```diff
@@ -31,23 +31,21 @@ interface ProfileProps {
 
 interface Props extends ProfileProps {
   logOutHandler: () => void;
-  setTheme: (theme: Themes) => void;
-  theme: Themes;
 }
 
-const profile: React.FC<Props> = ({
-  navigation,
-  logOutHandler,
-  setTheme,
-  theme,
-}) => {
+const profile: React.FC<Props> = ({navigation, logOutHandler}) => {
+  const {theme, setTheme} = useContext(ThemeContext);
```

Comment:
We should have HOC probably, that will allow us to pass props (theme) in component without import ThemeContext in all files.